controllers/storagecluster/external_resources.go

```diff
@@ -291,6 +302,37 @@ func (obj *ocsExternalResources) ensureCreated(r *StorageClusterReconciler, inst
 	return reconcile.Result{}, nil
 }
 
+func (r *StorageClusterReconciler) checkSCsAndCallCreateDefaultClaim(instance *ocsv1.StorageCluster) error {
+	storageClassList := &storagev1.StorageClassList{}
+	if err := r.Client.List(r.ctx, storageClassList); err != nil {
+		return fmt.Errorf("failed to list storageClasses. %v", err)
+	}
+
+	storageClass := &storagev1.StorageClass{}
+	for _, sc := range storageClassList.Items {
+		if strings.Contains(sc.Name, cephFsStorageClassName) ||
+			strings.Contains(sc.Name, cephRbdStorageClassName) {
+			storageClass = &sc
+		}
+	}
+
+	if storageClass.Name == "" {
+		return fmt.Errorf("storageClass for RBD and fileSystem not found")
+	}
+
+	annotations := storageClass.GetAnnotations()
+	if annotations == nil || annotations[StorageClassClaimAnnotation] == "" {
+		err := r.createDefaultStorageClassClaims(instance)
+		if err != nil {
+			return fmt.Errorf("failed to created storageClassClaim. %v", err)
+		}
+	} else {
+		r.Log.Info("storageClass with annotation `ocs.openshift.io.storagesclassclaim` already exist")
+	}
```

Comment:
Why do we need this message? it will appear on every reconciliation after the first one. 
Can we remove it?

Reply:
right, removed